packages/contracts/contracts/test/DAOMock.sol

```diff
@@ -4,27 +4,27 @@ pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 
-import "../core/acl/ACL.sol";
+import "../core/permission/PermissionManager.sol";
 import "../core/IDAO.sol";
 
-contract DAOMock is IDAO, ACL {
+contract DAOMock is IDAO, PermissionManager {
     mapping(uint256 => uint256) public totalSupply;
     mapping(address => mapping(uint256 => uint256)) public pastVotes;
 
     constructor(address initialOwner) initializer {
-        __ACL_init(initialOwner);
+        __PermissionManager_init(initialOwner);
```

Comment:
What's the difference between permission manager and permissions?

Reply:
Let's discuss this here https://github.com/aragon/core/pull/62#discussion_r928957492 and close this conversation.